pkg/lang/ir/types.go

```diff
@@ -81,3 +81,14 @@ type RunBuildCommand struct {
 	Commands  []string
 	MountHost bool
 }
+
+type AptConfig struct {
+	Name       string
+	Enabled    string
+	Types      string
+	URIs       string
```

Comment:
Will it store URI or multiple URIs?

If it just stores URI, please name it `URI`

Reply:
It could be multiple URIs for the same package (rare). According to the [spec](https://repolib.readthedocs.io/en/latest/deb822-format.html)

>The URI must specify the base of the Debian distribution archive, from which APT finds the information it needs. There must be a URI component present in order for the source to be valid; multipls URIs can be configured simultaneously by adding a space-separated list of URIs.

Reply:
LGTM